### PR TITLE
Link NewPipe icon to NewPipe SB and GPay to old Google Pay app

### PR DESCRIPTION
**Pull request by rodrigoswz**

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -54,6 +54,7 @@
     <icon drawable="@drawable/camera" package="org.lineageos.snap" name="Camera" />
     <icon drawable="@drawable/catogram" package="org.nift4.catox" name="Catogram" />
     <icon drawable="@drawable/catogram" package="ua.itaysonlab.messenger" name="Catogram" />
+    <icon drawable="@drawable/catogram" package="org.nift4.catox" name="Catogram" />
```
Comment (patrykmichalik) [inline app/src/main/res/xml/grayscale_icon_map.xml:57]: I would suggest removing this line as it is a duplicate of line 55.

```suggestion
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Hello! I would suggest removing this line as it is a duplicate of line 55.

**Comment by rodrigoswz** [comment]
Done, sorry about that 😅

**Comment by patrykmichalik** [comment]
No worries!

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
Thanks, @rodrigoswz!

**Pull request merged**